model/types.go

```diff
@@ -19,43 +19,52 @@ import (
 	"time"
 
 	"k8s.io/heapster/sinks/cache"
-	"k8s.io/heapster/store"
+	"k8s.io/heapster/store/daystore"
+	"k8s.io/heapster/store/statstore"
 )
 
 type Cluster interface {
 	// The Update operation populates the Cluster from a cache.
 	Update(cache.Cache) error
 
+	// The simple Get operations extract structural information from the Cluster.
+	GetAvailableMetrics() []string
+	GetNodes() []EntityListEntry
+	GetNamespaces() []EntityListEntry
```

Comment:
Ideally, only nodes and namespaces must be accessible from cluster. Those objects should then expose the pods that are under them. I am ok with this flat interface for now, but add a TODO.